Modules/sre_lib.h

```diff
@@ -453,7 +453,10 @@ do { \
     TRACE(("copy data to %p from %" PY_FORMAT_SIZE_T "d " \
            "(%" PY_FORMAT_SIZE_T "d)\n", \
            data, state->data_stack_base-size, size)); \
-    memcpy(data, state->data_stack+state->data_stack_base-size, size); \
+    /* We add an explicit cast to memcpy here because MSVC has a bug when
```

Comment:
I'm not sure that it's ok to put a comment into a #define macro. Maybe convert the macro to a static inline function, or put the comment outside (before) the #define.

Reply:
Good point, this works but it might lead to weird situations like

```c
/*
  some_commented_out_code();
  MACRO;
*/
```

and then the comment suddenly gets stopped in the middle due to macro expansion.